src/api/effect-scope.md

```diff
@@ -0,0 +1,59 @@
+# Effect Scope API <Badge text="3.2+" />
```

Comment:
Can you some use cases in this section? It is not clear enough in which situations the developers should use it

Reply:
See https://github.com/vuejs/docs/pull/1157/files#r682602039

The API reference only serves as a reference for the API signatures. Detailed usage will probably be discussed in a dedicated guide section later.